### ISSUE TITLE
[Chainquery] implement HEALTHCHECK 

### ISSUE THREAD

**Issue by Leopere** [lbryio/lbry-docker#59]
Eventually, we need boilerplate health checks for all of the containers shipped with the containers themselves.

**Comment by dangarthwaite** [comment]
I'd like to take this.  Although I believe the first healthcheck should be librycrd.
Is there something that can be curl'd in order to determine if chainquery [or librycrd] is ready?

**Comment by dangarthwaite** [comment]
I see this in Dockerfile:
```
## TODO: Implement healthcheck.
# HEALTHCHECK ["healthcheck"]
```

/usr/bin/healthcheck is a script that relies on curl, which isn't installed.  I'd like to take a stab getting it to work with lbrycrd-cli

**Comment by dangarthwaite** [comment]
I have a working patch for lbrycrd healthcheck.  Let's see if I can get chainquery done tonight.

**Comment by Leopere** [comment]
@dangarthwaite open some PR's!

**Comment by dangarthwaite** [comment]
So - the actual healthcheck for chainquery is untested.  A whole lot of other configuration files were tweaked to get lbrycrd running in regtest mode.  In the end this ticket is blocked until https://github.com/lbryio/chainquery/issues/121 is fixed.  `chainquery serve` gives an error when it starts.

 What does work are healthcheckers for lbrycrd and mysql containers that are in this PR.  Also a rewrite of chainquery/start.sh that ought to be wire-compatible and a LOT easier on the eyes.

**Comment by Leopere** [comment]
For the most part the Chainquery "healthcheck" in this case was just to make sure the process hadn't died and had completed it's initial boot process.

**Comment by dangarthwaite** [comment]
I changed chainquery's Dockerfile to download the latest version.  All three containers now have healthchecks.

```
dan@macbook:~/Projects/forks/lbry-docker
$ cd lbrycrd && docker-compose ps && cd ..
      Name          Command      State                                             Ports
------------------------------------------------------------------------------------------------------------------------------
lbrycrd_lbrycrd_1   start     Up (healthy)   11337/tcp, 0.0.0.0:11337->29245/tcp, 0.0.0.0:11336->29246/tcp, 9245/tcp, 9246/tcp
dan@macbook:~/Projects/forks/lbry-docker
$ cd chainquery && docker-compose ps && cd ..
         Name                       Command                State                     Ports
--------------------------------------------------------------------------------------------------------
chainquery_chainquery_1   start                         Up (healthy)   0.0.0.0:6300->6300/tcp
chainquery_mysql_1        docker-entrypoint.sh mysqld   Up (healthy)   0.0.0.0:3306->3306/tcp, 33060/tcp
```

**Comment by Leopere** [comment]
Fixed by #87
